homeassistant/components/light/mqtt_json.py

```diff
@@ -50,6 +61,10 @@
     vol.Optional(CONF_OPTIMISTIC, default=DEFAULT_OPTIMISTIC): cv.boolean,
     vol.Optional(CONF_BRIGHTNESS, default=DEFAULT_BRIGHTNESS): cv.boolean,
     vol.Optional(CONF_RGB, default=DEFAULT_RGB): cv.boolean,
+    vol.Optional(CONF_RGB_VALUE_TEMPLATE, default=DEFAULT_RGB_VALUE_TEMPLATE):
```

Comment:
Do not use a default value. Instead have it be `None` and add the `None` check back